notifier.go

```diff
@@ -15,37 +15,54 @@ import (
 	"time"
 )
 
+const defaultAirbrakeHost = "https://airbrake.io"
+
+func getCreateNoticeURL(host string, projectId int64, key string) string {
+	return fmt.Sprintf(
+		"%s/api/v3/projects/%d/notices?key=%s",
+		host, projectId, key,
+	)
+}
+
 type filter func(*Notice) *Notice
 
 var httpClient = &http.Client{
 	Transport: &http.Transport{
 		Proxy: http.ProxyFromEnvironment,
 		Dial: (&net.Dialer{
-			Timeout:   30 * time.Second,
+			Timeout:   15 * time.Second,
 			KeepAlive: 30 * time.Second,
 		}).Dial,
 		TLSHandshakeTimeout: 10 * time.Second,
 		TLSClientConfig: &tls.Config{
 			ClientSessionCache: tls.NewLRUClientSessionCache(1024),
 		},
-		MaxIdleConnsPerHost:   100,
+		MaxIdleConnsPerHost:   10,
```

Comment:
There are no good reasons except that it looks safer to me. Feel free to propose different number if you know how to configure it properly.